packages/office-ui-fabric-react/src/components/DatePicker/DatePicker.tsx

```diff
@@ -290,14 +290,21 @@ export class DatePicker extends BaseComponent<IDatePickerProps, IDatePickerState
   private _onSelectDate(date: Date) {
     const { formatDate, onSelectDate } = this.props;
 
+    if (this.props.calendarProps && this.props.calendarProps.onSelectDate) {
```

Comment:
did we have any instances where we were using this and it wasn't firing as expected? I can see that we override the original onSelectDate being passed to Calendar up above, just wanted to check if that was causing problems

Reply:
verified offline that this is expected and correct, and that both the sample page and calendar app are working with this change.